docs/_dg/UseCases.md

```diff
@@ -104,7 +104,7 @@ Extensions:
 
 #### Use Case 5: Find Item
 
-MSS:
+[[ MSS ]]:
```

Comment:
Like OS, this is an invalid key as the glossary entry name is "MSS (Main Success Scenario)". However, I'm going to change this to "Main Success Scenario (MSS)" in another PR, so let's fix it the right way.

```suggestion
[[ main-success-scenario-mss:MSS ]]:
```